e2e/ui/api-client.js

```diff
@@ -1,19 +1,41 @@
 import axios from "axios";
+import https from "https";
+import fs from "fs";
 
+export let NOMAD_PROXY_ADDR = process.env.NOMAD_PROXY_ADDR;
 export let NOMAD_ADDR = process.env.NOMAD_ADDR;
 export let NOMAD_TOKEN = process.env.NOMAD_TOKEN;
+export let NOMAD_CLIENT_CERT = process.env.NOMAD_CLIENT_CERT;
+export let NOMAD_CLIENT_KEY = process.env.NOMAD_CLIENT_KEY;
+export let NOMAD_CACERT = process.env.NOMAD_CACERT;
+export let NOMAD_TLS_SERVER_NAME = process.env.NOMAD_TLS_SERVER_NAME;
 
 if (NOMAD_ADDR == undefined || NOMAD_ADDR == "") {
   NOMAD_ADDR = "http://localhost:4646";
 }
 
+if (NOMAD_PROXY_ADDR == undefined || NOMAD_PROXY_ADDR == "") {
+  NOMAD_PROXY_ADDR = NOMAD_ADDR;
+}
+
 export const client = async (
   path,
   { data, method, headers: customHeaders, ...customConfig } = {},
   parameters
 ) => {
   const url = `${NOMAD_ADDR}/v1`.concat(path);
 
+  let httpsAgent = new https.Agent({keepAlive: true});
+
+  if (NOMAD_CLIENT_CERT !== "") {
```

Comment:
@tgross I did a poor job explaining the comment attached to this last time.

I was prodding that `NOMAD_CLIENT_CERT` could and would be `undefined` and if we want to compare based on both the case of an empty string `""` and `undefined` then we'd want this to read:

`if (!!NOMAD_CLIENT_CERT)`

These are quirks of JavaScript that I wouldn't expect anyone to know, however. I'll get a patch up for this.

Mea culpa. I'll be more direct and clear with PR comments in the future.

Reply:
Oh, thanks. I'd tested this out in a console but I think I might have just screwed up my testing because of the "double negative" of `("undefined !== "")`.

Reply:
I'll do better on pull requests feedbacks and comments in the future.

I follow this [Conventional Comments](https://conventionalcomments.org/) paradigm. However, I'll do better to be more explicit and with explanations... and less "prodding" comments.

Personally, I view PRs as part of the learning process hence prodding for more back and forth. But, I realize its not normal and untenable to have a lot of back and forth on PRs because we're a lean team.

I'll be better next time around.